components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -149,6 +149,7 @@ class PromptFeature(
      * @param grantResults The grant results for the corresponding permissions
      * @see [onNeedToRequestPermissions].
      */
+    @Suppress("UNUSED_PARAMETER")
```

Comment:
`permissions` is unused. Seems like misleading implementation. Should I raise issue on it?

Reply:
I believe the intent is to have a similar function signature to [`onRequestPermissionsResult`](https://developer.android.com/reference/android/support/v4/app/ActivityCompat.OnRequestPermissionsResultCallback).

Reply:
I think you are right. However parameter that is present in public API but absent in actual implementation can be potential cause of problems in the future. :)